ArduSub/system.cpp

```diff
@@ -97,7 +97,7 @@ void Sub::init_ardupilot()
 #if HAL_MOUNT_ENABLED
     // initialise camera mount
     camera_mount.init();
-    // This step ncessary so the servo is properly initialized
+    // This step is necessary so that servo is properly initialized
```

Comment:
I probably would have made this, "This step is necessary so that the servo is properly initialized"

Reply:
Updated the comment as per the suggestion.